wdl/transforms/new-base/src/main/scala/wdl/transforms/base/linking/expression/values/EngineFunctionEvaluators.scala

```diff
@@ -56,6 +56,20 @@ object EngineFunctionEvaluators {
     Try(Await.result(ioFunctionSet.readFile(fileToRead.value, Option(sizeLimit), failOnOverflow = true), ReadWaitTimeout))
   }
 
+  def split_lines(read: String): List[String] = {
+    if (read.nonEmpty) {
+      if (read.matches("[\\n]+")) {
+        val empty_string_list = List("")
+        println(empty_string_list)
+        empty_string_list
```

Comment:
Remove `println` and then you can just immediately return the value

```suggestion
        List("")
```